tools/bro-to-vast/bro-to-vast.cpp

```diff
@@ -330,7 +332,8 @@ int main(int argc, char** argv) {
     self->monitor(cmd.sink());
     self->receive(
       [&, query_id=query_id](const caf::down_msg&) {
-        std::cerr << "\ncompleted processing of query results" << std::endl;
+        std::cerr << std::endl; // newline after the dots.
```

Comment:
Why print the dots in the first place? Some form of activity indication? In this case, each dot should get printed with `std::flush`. Here, all dots get printed at once.

Reply:
Yeah, the `--show-progress` flags enables this form of activity indication. I fixed the buffering issue.